tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -538,7 +538,8 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
-		[ "$ARCH" == "ppc64le" ] && ([ "$LIBC" == "gnu" ] || die "LIBC type for ppc64le should be gnu")
+                # LIBC type for ppc64le should be gnu
```

Comment:
the following line is ensuring that the libc type for power is gnu. i'm not sure why you removed that die that will exit if this isn't true. as-is, this logic will not exit if it isn't, but rather just set it when it's checking it -- and i don't think that's what you meant to do. if it is, then, why didn't the original authors just set it here instead of checking and exiting if not? i'm guessing there was a reason (e.g. if it isn't set here it's not set overall, and this will cause an issue somewhere else that won't be fixed by just setting it here).

Reply:
By looking at [here](https://github.com/kata-containers/kata-containers/commit/02c1a59f6dd623713f5fe54f3bf4935fdd924336#diff-76e8e3f6645710d50d06912865744ee2228fb2fd0a5c694c65d6044e09d8db0d), thought it would be similar and better, that to set LIBC=gnu internally/dynamically instead of error and exit if LIBC is not set to gnu for ppc64le.

Reply:
Christy and Amulya,

I verified that building initrd image without LIBC specified at osbuilder dir failed.

```
ERROR: LIBC type for ppc64le should be gnu
make: *** [Makefile:87: /home/fedora/go/src/github.com/kata-containers/kata-containers/tools/osbuilder/.fedora_rootfs.done] Error 1
```

Reply:
@jing-wang4, it makes sense that the error is thrown since you didn't pass `LIBC=gnu` explicilty. However, what I was mentioning earlier was to try building the image by omitting the line entirely since the `LIBC` is being set here - https://github.com/kata-containers/kata-containers/blob/f09128d8c730cf1e1ce9c46133effc46d94bf1f0/src/agent/Makefile#L40

Reply:
Not sure if it is safe to remove it in this file.

Reply:
Thanks for confirming. So just a small cleanup as mentioned in the scrum this morning (just go ahead and set it), and you can submit this one upstream as well.

Reply:
ah, @Amulyam24 -- I think I see what you were saying now. That make target that @jing-wang4 is using is in the Makefile under tools/osbuilder. So unless I'm missing something, like there's a way to build the initrd (e.g. sudo -E PATH=$PATH make AGENT_INIT=yes DISTRO=fedora initrd) from the agent Makefile, this won't be set.

Reply:
Please see [update for commnet](https://github.com/jing-wang4/kata-containers/pull/6/commits/c8cf6d6bde426136437cdfec58009235d6effd71)